src/RazorSdk/Tool/Client.cs

```diff
@@ -171,9 +171,7 @@ public override async Task WaitForDisconnectAsync(CancellationToken cancellation
                     try
                     {
                         ServerLogger.Log($"Before poking pipe {Identifier}.");
-#pragma warning disable CA2022 // Avoid inexact read
-                        await Stream.ReadAsync(Array.Empty<byte>(), 0, 0, cancellationToken);
-#pragma warning restore CA2022
+                        _ = await Stream.ReadAsync(Array.Empty<byte>(), 0, 0, cancellationToken);
```

Comment:
Consider leaving comments explaining why we can't use ReadExactAsync so someone doesn't try to "fix" that in the future.

Reply:
Update: https://github.com/dotnet/sdk/pull/48766/commits/3137672c108c5e5405fb0ee717913be80ed17b6f